pom.xml

```diff
@@ -84,7 +84,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
             <version>1.6.1</version>
-            <scope>test</scope>
```

Comment:
Why?!
This dependency should only be in the top app layer whenever someone decides what bridge to use.